src/main.go

```diff
@@ -232,10 +318,10 @@ func process(data []models.DmwKnownContact, dfoAuthTokenObj models.DfoAuthTokenO
 		method := "processBatch"
```

Comment:
This should probably be declared outside of the loop since it doesn't appear to change.

Reply:
There are 2 functions inside the loop.  I added these in for logging.  Originally, I was just using the same variable name, "method" and changing it for the 2nd function.  I can't remember why I changed it to say method2 for the 2nd function.  I think there were issues with the logs.  I don't love the way I am logging and I know there is a much better way to do this but I need some help.  For now this works.